float64 goal_torque
---

